### PR TITLE
[App-2472] Update editorials UI

### PR DESCRIPTION
**Pull request by ricardo-frango**
**What does this PR do?**

   Update Editorials UI

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] EditorialViewScreen.kt
- [ ] EditorialView.kt

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APP-2472](https://aptoide.atlassian.net/browse/APP-2472)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2472](https://aptoide.atlassian.net/browse/APP-2472)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[APP-2472]: https://aptoide.atlassian.net/browse/APP-2472?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2472]: https://aptoide.atlassian.net/browse/APP-2472?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by G0retZ — CHANGES_REQUESTED** [review]

File: app-games/src/main/java/com/aptoide/android/aptoidegames/editorial/EditorialView.kt [hunk 1]
```diff
@@ -105,114 +110,65 @@ fun EditorialBundle(
...
-@Composable
-fun EditorialsViewCardPreview() {
-  AptoideTheme(darkTheme = isSystemInDarkTheme()) {
-    EditorialsViewCard(
-      articleMeta = randomArticleMeta,
-      onClick = {}
-    )
-  }
-}
+// Components
```
[the remark this answers is not in the document]
Comment (ricardo-frango) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/editorial/EditorialView.kt]: it was already there. just forgot to remove. fixed

File: app-games/src/main/java/com/aptoide/android/aptoidegames/editorial/EditorialView.kt [hunk 2]
```diff
@@ -105,114 +110,65 @@ fun EditorialBundle(
...
+    )
+    Text(
+      text = articleMeta.caption,
+      style = AppTheme.typography.body,
+      color = AppTheme.colors.primary,
+      maxLines = 1,
+      overflow = TextOverflow.Ellipsis,
+      modifier = Modifier
+        .padding(start = 8.dp, top = 8.dp)
+        .background(color = AppTheme.colors.onPrimary)
```
[the remark this answers is not in the document]
Comment (ricardo-frango) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/editorial/EditorialView.kt]: fixed

File: app-games/src/main/java/com/aptoide/android/aptoidegames/editorial/EditorialView.kt [hunk 3]
```diff
@@ -105,114 +116,63 @@ fun EditorialBundle(
...
-@Composable
-fun EditorialSummary(getSummary: () -> String) {
   Text(
-    text = getSummary(),
+    modifier = modifier,
+    text = articleMeta.title,
+    maxLines = 1,
+    overflow = TextOverflow.Ellipsis,
+    style = AppTheme.typography.inputs_L,
+    color = pureWhite
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/editorial/EditorialView.kt:155]: Lets use the new colors 😄

**Review by G0retZ — CHANGES_REQUESTED** [review]

File: app-games/src/main/java/com/aptoide/android/aptoidegames/editorial/EditorialView.kt (hunk 3 above)
Comment (ricardo-frango) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/editorial/EditorialView.kt]: fixed :D

**Review by G0retZ — APPROVED** [review]

**Pull request merged**